grails-app/views/tutorials/manage.gsp

```diff
@@ -37,7 +37,7 @@
             Add Tutorial
         </a>
         <cl:ifSiteAdmin>
-        <g:if test="${params.migrate}">
+        <g:if test="${params.admin}">
```

Comment:
This might be by lack of familiarity with the system/intent here but admin status doesn't seem like something that should be trusted as an HTTP parameter.  Is there any way a non-admin could get the admin version of the filter by just appending this param to the URL?

Reply:
This is not admin status/auth. This is to view the administrative tutorials. Possibly bad choice in keyword.

Reply:
Thanks.  The ifSiteAdmin check below that adds the parameter was part of my reasoning - assuming it's just that admins are the ones likely to want to view these?

Reply:
That's correct.